src/box/lua/schema.lua

```diff
@@ -1981,6 +1981,8 @@ local function check_pairs_opts(opts, key_is_nil)
     return iterator, after
 end
 
+box.internal.check_pairs_opts = check_pairs_opts -- for net.box
```

Comment:
Nit: `net.box` doesn't use this function.

Reply:
Fixed